Doppler.ContactPolicies.Data.Access/Entities/ContactPoliciesSettings.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Data.Access.Entities
+{
+    public sealed class ContactPoliciesSettings
+    {
+
+        public User User { get; set; }
```

Comment:
Is this field useful?

Reply:
kind of:
`


<!--StartFragment-->result = contactPoliciesSettingsEnumerable.GroupBy(ct =&gt; ct.User.IdUser).Select(g =&gt;                {                    var groupedPost = g.First();                    groupedPost.SubscribersLists = g.Select(ct =&gt; ct.SubscribersLists.Single()).ToList();                    return groupedPost;                }).First();<!--EndFragment-->

`

Reply:
I am not sure yet. Let's talk about it.

Reply:
Creo que este comentario aplica acá: 

> The User class is used to filter the accountName that is passed to me as a parameter

Esta clase `ContactPoliciesSettings`, es la clase representa los datos de salida de la API (aunque está en el namespace `Doppler.ContactPolicies.Data.Access.Entities` 🤔), entonces, realmente necesitamos el `User` o es solo una cuestión de implementación y en ese caso no debería estar expuesto.